src/main/resources/application.yaml

```diff
@@ -1,3 +1,15 @@
 bot:
   token: ${BOT_TOKEN}
   name: ${BOT_NAME}
+  answers:
```

Comment:
@l3r8yJ Maybe we should store message content in `properties`, I think application.yml is designed for config only